include/bluetooth/mesh/cdb.h

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2019 Tobias Svehagen
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_BLUETOOTH_MESH_CDB_H_
+#define ZEPHYR_INCLUDE_BLUETOOTH_MESH_CDB_H_
+
+#if defined(CONFIG_BT_MESH_CDB)
+#define NODE_COUNT    CONFIG_BT_MESH_CDB_NODE_COUNT
+#define SUBNET_COUNT  CONFIG_BT_MESH_CDB_SUBNET_COUNT
+#define APP_KEY_COUNT CONFIG_BT_MESH_CDB_APP_KEY_COUNT
+#else
+#define NODE_COUNT    0
+#define SUBNET_COUNT  0
+#define APP_KEY_COUNT 0
+#endif
+
+enum {
+	BT_MESH_CDB_NODE_CONFIGURED,
+	BT_MESH_CDB_NODE_BLACKLISTED,
+
+	BT_MESH_CDB_NODE_FLAG_COUNT
+};
+
+struct bt_mesh_cdb_node {
+	u8_t  uuid[16];
+	u16_t addr;
+	u16_t net_idx;
+	u8_t  num_elem;
+	u8_t  dev_key[16];
+
+	ATOMIC_DEFINE(flags, BT_MESH_CDB_NODE_FLAG_COUNT);
+};
+
+struct bt_mesh_cdb_subnet {
+	u16_t net_idx;
+
+	bool kr_flag;
+	u8_t kr_phase;
+
+	struct {
+		u8_t net_key[16];
+	} keys[2];
+};
+
+struct bt_mesh_cdb_app_key {
+	u16_t net_idx;
+	u16_t app_idx;
+
+	struct {
+		u8_t app_key[16];
+	} keys[2];
+};
+
+enum {
+	BT_MESH_CDB_VALID,
+	BT_MESH_CDB_SUBNET_PENDING,
+	BT_MESH_CDB_KEYS_PENDING,
+	BT_MESH_CDB_NODES_PENDING,
+
+	BT_MESH_CDB_FLAG_COUNT,
+};
+
+struct bt_mesh_cdb_net {
```

Comment:
Is the `_net` really necessary here? A CBD instance is implicitly for a single Mesh network. You also haven't bothered appending `_net` for the global variable name.